redisvl/schema/schema.py

```diff
@@ -170,12 +170,12 @@ def _make_field(storage_type, **field_inputs) -> BaseField:
 
     @root_validator(pre=True)
     @classmethod
-    def validate_and_create_fields(cls, values):
+    def validate_and_create_ields(cls, values: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def validate_and_create_fields(cls, values: Dict[str, Any]) -> Dict[str, Any]:
```

Reply:
Oh sorry I think I might have jumped on this too soon since it's still draft ;) but just heads up

Reply:
Hah, good catch 😂